src/routes/breaddit/breaddit.controller.js

```diff
@@ -0,0 +1,104 @@
+const fs = require("fs");
+const { promisify } = require("util");
+const { validationResult } = require("express-validator");
+
+const breadditsData = require("../../../db/dummy.data.json");
+const data = breadditsData.breaddits;
+
+const writeFile = promisify(fs.writeFile);
+
+const listBreaddits = (req, res) => {
+  return res.json({
+    data
+  });
+};
+
+const createBreaddit = async (req, res) => {
+  // console.log('body', req.body)
```

Comment:
Don't forget to remove commented-out code! I usually try not to commit it, unless it's accompanied by a comment about what it's for.